skada/deep/tests/test_deep_divergence.py

```diff
@@ -195,3 +197,20 @@ def test_can_with_custom_callbacks():
     callback_classes = [cb.__class__.__name__ for cb in method.callbacks]
     assert "EpochScoring" in callback_classes
     assert "ComputeSourceCentroids" in callback_classes
+
+
+def test_dan_loss_edge_cases():
+    # Create identical source features to get median distance = 0
+    features_s = torch.tensor([[1.0, 2.0], [1.0, 2.0]], dtype=torch.float32)
+    features_t = torch.tensor([[3.0, 4.0], [5.0, 6.0]], dtype=torch.float32)
+
+    # Verify median distance is 0
+    assert torch.median(torch.cdist(features_s, features_s)) == 0
+
+    # Test that dan_loss still works
+    loss = dan_loss(features_s, features_t)
```

Comment:
for me the median will not be 0 here no ?

Reply:
In `dan_loss` we compute:
`median_pairwise_distance = torch.median(torch.cdist(features_s, features_s))`
And we checked above that's its equal to 0

Reply:
oh okay! perfect